source/_includes/asides/docs_navigation.html

```diff
@@ -40,16 +40,8 @@ <h1 class="title delta">Topics</h1>
         </ul>
       </li>
       <li>
-        <b>{% active_link /docs/assist/ Assist %}</b>
+        <b>{% active_link /voice/ Assist %}</b>
```

Comment:
We need to be consistent in our naming. If top-level is called "Voice Control", should it be called "Voice Control" here too ?